packages/tuxguitar/PKGBUILD

```diff
@@ -0,0 +1,127 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Stephan Springer <buzo+arch@Lini.de>
+# Contributor: Thanos Apostolou <thanosapostolou@outlook.com>
+# Contributor: Alexandre Moine <alexandre@moine.me>
+# Contributor: Sergej Pupykin <pupykin.s+arch@gmail.com>
+# Contributor: |AhIoRoS| < ahioros@gmail.com >
+# Contributor: Florian Hülsmann <fh@cbix.de>
+
+pkgbase=tuxguitar
+pkgname=tuxguitar
+pkgver=1.5.5
+pkgrel=2
+pkgdesc='Multitrack guitar tablature editor and player'
+arch=(x86_64 i686 armv7h)
```

Comment:
What about aarch64? Or conversely, why include ARM architectures at all? Are we able to produce ARM binary packages?

Reply:
TG provides build targets for these three architectures (they were also included in the original AUR version). I could probably write another patch to allow building for aarch64 but I'd prefer to "wait for upstream"  for official support.

In general, I can already build + check aarch64 packages ([mamba builds out of the box](https://ci.cbix.de/osam-cologne/archlinux-proaudio/57/2/3)) so we could easily provide basic 64-bit ARM support in our repo as well for packages that successfully build. Personally ARM is something I'd like to support, even though the experience with Arch is currently not fully comparable. Projects like "use your RasPi with Archlinux ARM as a MIDI synth" are definitely something I can imagine some people to consider 😛

(I still keep a RasPi 2 around somewhere so we could even build armv7h without cross compilation, but it will probably lose relevance rather soon)

Reply:
Well ok I just tried building for armv7hl and it failed because the SWT version doesn't exist for that architecture. I'll remove armv7h for now.